tests/run_magetab_converter.py

```diff
@@ -0,0 +1,148 @@
+
+from converter.converting import *
+from converter.datamodel import SeqAssay, Protocol, MicroarrayAssay, DataFile, AssayData, Study, Project, Sample
+from converter.parsing import parse_sdrf, parse_idf
+from utils.converter_utils import strip_extension, write_json_file
+
+wd = dir_path = os.path.dirname(os.path.realpath(__file__))
+idf_path = os.path.join(wd, 'test_data', 'E-MTAB-6160.idf.txt')
```

Comment:
I would suggest that the accessions could be passed as arguments and then you could have some logic on travis to try various accessions. Then everytime you find an issue with a new accession and you fix it, you can add then a new test example to be tested on every PR.